docs/test.md

```diff
@@ -0,0 +1,3 @@
+# Notes on the model testing stage
+The `testing/main.py` script runs the model testing stage. You select a trained
+model and a region (which should be new/unseen) to test it on.
```

Comment:
Can more info be added on how to select model and region, what the test does, and what the expected output is?

Reply:
The code for selecting what data to test with is a little messy, but most happens here:

https://github.com/m2lines/gz21_ocean_momentum/blob/2ea18f57a57c6011fdbbc9dcb3510ff39208fdef/src/gz21_ocean_momentum/testing/main.py#L192-L205

It's not configurable as-is. It also takes a bunch of params from previous stage runs, which should impact region selection.

I think we keep this entry lean for now, until we tackle the testing stage.